### PR TITLE
Energy Chainsaws

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: Added the Energy Chainsaw to the traitor uplink for Bartender, Botanist and Chef.
/:cl:

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
gimmie pictures!!

**Comment by PopNotes** [comment]
One sec boss

**Comment by PopNotes** [comment]
4u

File: hippiestation/code/game/objects/items/twohanded.dm [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	lefthand_file = 'hippiestation/icons/mob/inhands/lefthand.dmi'
+	righthand_file = 'hippiestation/icons/mob/inhands/righthand.dmi'
+	force_on = 60 //I'VE GONE COMPLETELY INSANE! HA HA HA HA!
+	w_class = WEIGHT_CLASS_HUGE
+	origin_tech = "materials=5;engineering=4;combat=4;syndicate=4"
+	attack_verb = list("sawed", "shred", "rended", "gutted", "eviscerated")
+	actions_types = list(/datum/action/item_action/startchainsaw)
+	block_chance = 50
+	armour_penetration = 15
+	var/onsound
```
Comment (ghost) [inline hippiestation/code/game/objects/items/twohanded.dm:15]: why split the definitions up?

**Comment by McBawbaggings** [comment]
i think the doom comic comment alone is enough to support this code
+1

File: hippiestation/code/game/objects/items/twohanded.dm (hunk 1 above)
Comment (PopNotes) [reply hippiestation/code/game/objects/items/twohanded.dm]: If I don't, picking up/dropping the chainsaw makes the revving up/down sounds, which I don't want.

**Pull request merged**